src/rollup/BridgeCreator.sol

```diff
@@ -96,7 +97,11 @@ contract BridgeCreator is Ownable {
         } else {
             IERC20Bridge(address(frame.bridge)).initialize(IOwnable(rollup), nativeToken);
         }
-        frame.sequencerInbox.initialize(IBridge(frame.bridge), maxTimeVariation);
+        frame.sequencerInbox.initialize(
+            IBridge(frame.bridge),
```

Comment:
same here, if we want avail to be an opt-in service, this should be optional and abstracted away